### PR TITLE
Adopt buildkites format

### PR DESCRIPTION
**Pull request by encee**
Moves from `source -> dest` to `[from: source, to: dest]`

### CODE REVIEW

**Review by toadzky — APPROVED** [review]
i don't really like the complex conditionals that are all just chained together. i find bash conditionals hard to follow, so having several in a row that all seem to be checking almost the same thing is hard to read. can we pull the comparisons into variables so it's easier to read?

**Comment by encee** [comment]
Done.

**Review by toadzky — APPROVED** [review]
mucho mas readable
